app/javascript/dashboard/modules/contact/components/ContactDropdownItem.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="option-item--user">
+    <thumbnail :src="thumbnail" size="24px" :username="name" />
```

Comment:
The `<thumbnail>` component is not registered correctly.

![Screenshot 2021-06-29 at 11 34 19 AM](https://user-images.githubusercontent.com/12408980/123745467-f4176000-d8cd-11eb-89b2-1722aecac59f.png)